HomeAssistant/Views/SettingsDetailViewController.swift

```diff
@@ -781,6 +845,172 @@ class SettingsDetailViewController: FormViewController, TypedRowControllerType {
             }
         }
     }
+
+    func getSerialNumberForAccessory(_ accessory: HMAccessory) -> String? {
+        let infoService = accessory.services.first { $0.serviceType == HMServiceTypeAccessoryInformation }
+
+        let serialNumberChar = infoService?.characteristics.first {
+            $0.characteristicType == "00000030-0000-1000-8000-0026BB765291"
```

Comment:
`HMCharacteristicTypeSerialNumber` looks deprecated, is there another way to get this info?

Reply:
I checked and couldn’t find anything better sadly.